### DISCUSSION TITLE
Nemo and archive files

### DISCUSSION THREAD

**Discussion by PRSoftware** [linuxmint/discussions#136] [Pull requests]
Hello,

I switched to Linux Mint two years ago after many many years on Windows.

One useful thing on Windows is the management of the archive files in the file explorer.

They are seen as simple folders: you can navigate inside, add and remove files or directories directly in the file explorer.

In Nemo on Linux Mint, opening an archive files opens a new window in the archive manager.

I find it not useful and counter productive because you must change the current window, manage the content of the archive file and close the archive manager before continuing working in the file explorer. It is really easier to work inside Nemo without switching to another window.

Is it a way (parameter or plugin) to manage the most popular formats of archive files (7z, bz2, tar, tgz, zip) directly in Nemo ?

If not, is it planned in a nearby release ? When ?

If not, could it be planned for a future version ?

Patrick

**Comment by HT-7** [comment]
Nemo should at least be able to open ZIP files. For creation, file-roller is sufficient.

Also the stock Android file manager opens ZIP files but it can not create them. Strangely, Samsung's "My Files" application can not browse and only extract and create ZIP files.
